randovania/games/samus_returns/logic_database/header.json

```diff
@@ -1049,25 +1049,25 @@
             "Knowledge": {
                 "long_name": "Knowledge",
                 "description": "Some destructible objects have vulnerabilities other than those which the player is informed of. For example, Power Bomb can be used to open charge beam doors. Enabling this trick may require players to apply this knowledge in order to progress.",
-                "require_documentation_above": 0,
+                "require_documentation_above": 2,
```

Comment:
whats the reasoning for this? IMO all knoweldge tricks should have a comment attached

Reply:
Probably not thinking. Will revert.